app/src/main/java/com/example/warehousemanagement/domain/model/ImportPackages.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.warehousemanagement.domain.model
+
+import java.util.Date
+
+data class ImportPackages(
+    val id: String,                // Định danh gói nhập
+    val packageName: String,        // Tên gói nhập
+    val importDate: Date,           // Ngày nhập hàng
+    val supplier: Supplier,         // Thông tin nhà cung cấp
+    val idReceivedBy: String,       // Người nhận hàng
+    val status: Status= Status.PENDING,  // Tình trạng gói nhập, default is PENDING
+    val note: String? = null        // Ghi chú, optional
+)
```

Comment:
After checked by worker, if there're some deficit ( thâm hụt) , remember it nha

Reply:
when there's some deficit, this package will be canceled

Reply:
No, we use it to report with supplier to get return money or sth else. Those packages are alread delived to  our warehouse, and i thinks all package will have deficit.